sdk/core/azure-core-management/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0 (2020-09-24)
 
 - Updated class method names.
```

Comment:
Do consider breaking the updates into following sections https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change

Reply:
Thanks for the info. Get the idea, minor bug fix does not need get into CHANGELOG, and major interface change and feature support will need be there in CHANGELOG.